### PR TITLE
actually generate TS schema from GraphQL

### PR DESCRIPTION
**Pull request by rvantonder**
After a long time debugging, I couldn't understand why new GQL definitions didn't show up in `schema.ts`. I realized a `schema.ts` is being generated, but not where the current one lives. 

If we moved it under `src/graphql`, we should probably be generating it there. I updated the script and after `yarn generate`, the `schema.ts` is updated with a lot of things that were missing before... (and probably unused, but nevertheless, missing).

addendum: apparently the build looks for schema.ts under src, but it still doesn't keep the one in graphql in sync

## Test plan
Generates file as expected, dev experience issue.

### CODE REVIEW

File: client/shared/gulpfile.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ async function graphQlSchema() {
         postProcessor: code => format(code, { ...formatOptions, parser: 'typescript' }),
       }
     )
-  await writeFile(__dirname + '/src/schema.ts', typings)
+  await writeFile(__dirname + '/src/graphql/schema.ts', typings)
```
Comment (rvantonder) [inline client/shared/gulpfile.js:65]: The previous `/src/schema.ts` file [does not exist](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:client/shared/src/schema.ts&patternType=literal) so something is not right here

**Comment by felixfbecker** [comment]
The file should be `.gitignored` and only generated on build (i.e. it shouldn't be committed like in this PR). There is probably a gitignore entry for the old path

**Comment by rvantonder** [comment]
OK, sounds like I need to gitignore https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/shared/src/graphql/schema.ts?subtree=true then

**Comment by rvantonder** [comment]
Or, rather, that file shouldn't exist?

**Pull request closed without merging**